package.json

```diff
@@ -70,6 +70,7 @@
   "devDependencies": {
     "@biomejs/biome": "1.5.3",
     "@keplr-wallet/types": "0.12.23",
+    "@radix-ui/react-slider": "1.1.2",
```

Comment:
[radix-ui slider](https://www.radix-ui.com/primitives/docs/components/slider) replaces native `<input type="range"/>`  with lots of css overrides to achieve style requirement